frontend/public/components/modals/add-secret-to-workload.tsx

```diff
@@ -7,13 +7,15 @@ import { DeploymentModel, DeploymentConfigModel, StatefulSetModel } from '../../
 import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory/modal';
 import { Dropdown, history, ResourceIcon, ResourceName, resourcePathFromModel } from '../utils';
 import { RadioInput } from '../radio';
+/* eslint-disable import/named */
```

Comment:
I didn't think this was still needed after we updated the i18next dependencies. Can you check to see if you still see import errors?

Reply:
Hi Sam,
i still got error when commit the change without the comment

Reply:
> i still got error when commit the change without the comment

Can you paste the error?

Reply:
here is the pre commit error:
```
husky > pre-commit (node v10.19.0)
✔ Preparing...
⚠ Running tasks...
  ❯ Running tasks for *.{js,jsx,ts,tsx,json,gql,graphql}
    ✖ eslint --color --fix [FAILED]
↓ Skipped because of errors from tasks. [SKIPPED]
✔ Reverting to original state because of errors...
✔ Cleaning up... 

✖ eslint --color --fix:

/Users/jingjingpan/git/openshift/jjpan/console/frontend/public/components/modals/add-secret-to-workload.tsx
  11:34  error  WithTranslation not found in 'react-i18next'  import/named

✖ 1 problem (1 error, 0 warnings)

husky > pre-commit hook failed (add --no-verify to bypass)
```

Reply:
OK, it looks like this is a known issue specifically with eslint and `WithTranslation`. We can leave `eslint-disable`